packages/ui/Scrollable/Scrollable.js

```diff
@@ -785,18 +826,57 @@ class ScrollableBase extends Component {
 		forward(type, {scrollLeft: this.scrollLeft, scrollTop: this.scrollTop, moreInfo: this.getMoreInfo()}, this.props);
 	}
 
+	compareAndSet (obj, name, value) {
+		const changed = (obj[name] !== value);
+		obj[name] = value;
+		return changed;
+	}
+
+	forwardReachEdgeEvent (orientation) {
```

Comment:
Could you suggest another function name because this function does not always forward an event? On the other hand, `forwardScrollEvent ` function always send an event.